contracts/GateSealFactory.vy

```diff
@@ -0,0 +1,39 @@
+# @version 0.3.7
+
+event GateSealCreated:
+    gate_seal: address
+
+EIP5202_CODE_OFFSET: constant(uint256) = 3
```

Comment:
need to add some references and mention the particular preamble that is expected